app/views/devise/registrations/edit.html.erb

```diff
@@ -28,7 +28,7 @@
     <%= f.password_field :current_password, autocomplete: "off" %>
   </div>
 
-  <div class="actions">
+  <div class="actions", style="margin-top: 10px;">
```

Comment:
There shouldn't be a comma in an HTML element.  Also, instead of putting the style directly in the HTML, it might make sense to give the HTML element a class and using a CSS file to add the property.